Assets/PatchKit Patcher/Scripts/AppData/Remote/RemoteMetaData.cs

```diff
@@ -95,5 +95,17 @@ public string GetKeySecret(string key, string cachedKeySecret)
 
             return keySecret;
         }
+
+        public AppVersion GetAppVersionInfo(int versionId)
+        {
+            if (versionId < 0)
```

Comment:
As I remember, version id cannot be zero. @genail am I right?

Reply:
Most probably you're right. I can't recall any scenario where version is 0. When a version is not published, it just returns 404.